frontend/components/layout/ElectronUpdateBanner.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+    <div class="w-full py-2 text-center bg-red-400">
+        <h1>
+            A new version (v{{ version }}) is available for download at
+            <a class="underline" :href="latestReleaseUrl" target="_blank">
+                {{ latestReleaseUrl }}
+            </a>
+        </h1>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { repository } from '~/package.json';
```

Comment:
This value of the `repository.url` in the `frontend/package.json` can be different from the value of the `electron/package.json` that is actually checked by the electron updater, please use correct one